hashes/src/sha256t.rs

```diff
@@ -244,7 +244,7 @@ macro_rules! sha256t_hash_newtype {
             /// Hashes the entire contents of the `reader`.
             #[cfg(feature = "bitcoin-io")]
             #[allow(unused)] // the user of macro may not need this
-            fn hash_reader<R: io::BufRead>(reader: &mut R) -> Result<Self, io::Error> {
+            fn hash_reader<R: $crate::io::BufRead>(reader: &mut R) -> Result<Self, $crate::io::Error> {
```

Comment:
Per #3135 this method shouldn't exist in the first place. Maybe we can address that first so that we don't end up with funny commits changing things and then removing them?

Reply:
Now removed in #3200